scripts/cmake/CCacheSetup.cmake

```diff
@@ -20,7 +20,7 @@ set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
 set_property(GLOBAL PROPERTY RULE_LAUNCH_LINK ccache)
 
 if(COMPILER_IS_CLANG)
-    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Qunused-arguments")
+    add_compile_options(-Qunused-arguments)
```

Comment:
@endJunction ?

Reply:
From the [clang docu](https://clang.llvm.org/docs/ClangCommandLineReference.html)
```-Qunused-arguments     Don’t emit warning for unused driver arguments```

I guess it is silencing the ccache options not used by clang. It was added [here](https://github.com/ufz/ogs/commit/b64ba652ec3919d8ff540c8cebebe80839f7ca1e#diff-514117eda07d1908176952bc79044fb9R54), isn't it ;) ?

Feel free to remove the option, if there are not too many warnings because of unused ccache options.